packages/peregrine/lib/store/actions/cart/asyncActions.js

```diff
@@ -108,14 +92,6 @@ export const addItemToCart = (payload = {}) => {
                     })
                 );
 
-                // and fetch details
-                await dispatch(
-                    getCartDetails({
```

Comment:
Do we need to fetch details on errors anymore? We remove/create cart so the state should be reset to empty anyways. I removed this from the relevant actions.